shared/versions/stack/master.asciidoc

```diff
@@ -1,17 +1,17 @@
-:version:                8.16.0
+:version:                9.0.0
```

Comment:
Should this actually be alpha1?

```suggestion
:version:                9.0.0-alpha1
```

Reply:
Yeah, it's likely but I can't tell yet from looking on local QA deployments or artifacts.elastic.co, so I'll put this PR into draft state for now.

Reply:
Oklahoma